c/httpserver.c

```diff
@@ -209,6 +209,14 @@ static char crlf[] ={ 0x0d, 0x0a};
 #define DEFAULT_UMASK 0022
 #endif
 
+//No need to fill logs with the same message, warn based on a desired frequency
+#ifdef __ZOWE_OS_ZOS
+#ifndef TLS_WARN_FREQUENCY
+#define TLS_WARN_FREQUENCY 25
+#endif
+static int tlsWarnCounter = 0;
```

Comment:
Please use `unsigned int`, otherwise if we ever hit integer overflow, it will result in undefined behavior.